packages/e2e/example/test_driver/failure.dart

```diff
@@ -21,7 +21,10 @@ void main() {
 
   testWidgets('failure 1', (WidgetTester tester) async {
     // Build our app and trigger a frame.
-    await tester.pumpWidget(MyApp());
+    app.main();
+
+    // Trigger a frame.
+    await tester.pumpAndSettle();
```

Comment:
Would it be possible to `await expectLater` below instead of calling this?

I worry about littering tests with `tester.pumpAndSettle()`, it seems kind of mysterious whether it's needed or not. In any case, I hope that `WidgetTester` synchronization will help.

Reply:
done.